tests/dialects/test_snowflake.py

```diff
@@ -19,6 +19,7 @@ def test_snowflake(self):
         self.assertEqual(ast.sql("snowflake"), "DATEADD(MONTH, n, d)")
 
         self.validate_identity("SELECT GET(a, b)")
+        self.validate_identity("SELECT HASH_AGG(amount)")
```

Comment:
Let's also include tests with more than 1 arguments to validate the var len args:



```suggestion
        self.validate_identity("SELECT HASH_AGG(a, b, c, d)")
```